apps/src/Convert.cpp

```diff
@@ -67,55 +60,46 @@ int main(int argc, char* argv[])
         return EXIT_FAILURE;
     }
 
-    // Get the output options
-    outputPath = parsedOptions["output-file"].as<std::string>();
-    auto depth = static_cast<BitDepth>(parsedOptions["bit-depth"].as<int>());
+    // Parse the cmd line
+    imgPath = parsedOptions["input-path"].as<std::string>();
+    if (!fs::exists(imgPath)) {
+        std::cerr << "ERROR: Input file does not exist" << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    outputPath = parsedOptions["output-path"].as<std::string>();
```

Comment:
For the depths, it can actually be done with just two comparisons: `(r.depth() == g.depth()) && (r.depth() == b.depth())` The result of that is transitively the same as `g.depth() == b.depth()`